exporters/zipkin/src/main/java/io/opentelemetry/exporter/zipkin/ZipkinSpanExporterBuilder.java

```diff
@@ -21,6 +23,7 @@
 /** Builder class for {@link ZipkinSpanExporter}. */
 public final class ZipkinSpanExporterBuilder {
   private BytesEncoder<Span> encoder = SpanBytesEncoder.JSON_V2;
+  private Supplier<InetAddress> localIpAddressSupplier = LocalInetAddressSupplier.INSTANCE;
```

Comment:
#nit: to be consistent with other singletons, let's make `LocalInetAddressSupplier.INSTANCE` private and access it via `LocalInetAddressSupplier#getInstance()`.